ebos/eclproblem.hh

```diff
@@ -2684,31 +2699,27 @@ private:
     }
 
     // this method applies the runtime constraints specified via the deck and/or command
-    // line parameters for the size of the next time step size.
+    // line parameters for the size of the next time step.
     Scalar limitNextTimeStepSize_(Scalar dtNext) const
     {
-        const auto& events = this->simulator().vanguard().schedule().getEvents();
-        int episodeIdx = this->simulator().episodeIndex();
+        if (!enableExperiments)
+            return dtNext;
+
```

Comment:
Using !enableExperiments for flow spesific stuff is in my opinion not a good strategy. Or have I misunderstood this line?

Reply:
the problem is that this is called deeply within `flow` and it causes the `remainingEpisodeTime` to be smaller than 0 at some occasions. Since the result for is ignored for `flow` anyway, I gave up on fixing this. This statement applies for a work-in-progress version of the patch, so maybe it is not required anymore...